protocol/testing/e2e/gov/bridge_test.go

```diff
@@ -33,7 +35,18 @@ func TestUpdateEventParams(t *testing.T) {
 			},
 			expectedProposalStatus: govtypesv1.ProposalStatus_PROPOSAL_STATUS_PASSED,
 		},
-		"Fail: invalid authority": {
+		"Failure: empty eth address": {
+			msg: &bridgetypes.MsgUpdateEventParams{
+				Authority: authtypes.NewModuleAddress(govtypes.ModuleName).String(),
+				Params: bridgetypes.EventParams{
+					Denom:      genesisEventParams.Denom,
```

Comment:
should this denom be `genesisEventParams.Denom + "updated"`?

Reply:
Are you using this to validate that the denomination was not updated? If so, would you mind creating a constant 'UpdatedDenom`, maybe also `UpdatedEthChainId`, at the top of the file?

Reply:
good call! let me create a genesis variable